core/src/i18n/en.yaml

```diff
@@ -46,12 +46,34 @@ cluster-role-bindings:
 cluster-roles:
   title: ClusterRoles
 clusters:
+  add:
+    title: Add Cluster
+  common:
+    api-server-address: API server address
+    download-kubeconfig: Download kubeconfig
+    kubeconfig-download-error: Failed to download the Kubeconfig
+    kubeconfig-not-present: Kubeconfig is missing on the Cluster
   list:
     outdated:
       download: Download Kubeconfig
       tooltip: Outdated parameter version may cause errors. Delete and re-add your cluster.
+    no-clusters-found: No clusters found
   overview:
     title: Cluster Overview
+    version: Version
+  storage:
+    changed-message: Cluster storage changed from {{originalStorage}} to {{targetStorage}}.
+    choose-storage:
+      label: Configuration storage type
+    descriptions:
+      localStorage: Cluster data is persisted between browser reloads.
+      sessionStorage: Cluster data is cleared when the page session ends.
+      inMemory: Cluster data is cleared when the page is refreshed or navigated out of.
```

Comment:
please sort the yaml files. (you can try https://marketplace.visualstudio.com/items?itemName=PascalReitermann93.vscode-yaml-sort )

Reply:
Damnn